adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/persitence/mongo/MongoClient.kt

```diff
@@ -42,7 +42,7 @@ class MongoClient {
         var uri = if (username != null && password != null) {
             "mongodb://$username:$password@$host:$port/$dbName"
         } else {
-            "mongodb://$host:$port"
+            "mongodb://$host:$port/?serverSelectionTimeoutMS=100"
```

Comment:
If you extract that number to a named const it can also double down as documentation as to why you chose that number. LOCAL_LOOPBACK_TIMEOUT_FOR_DEVELOPMENT or something like that

Bonus points if this number is extracted to a properties file

Reply:
will do - was unsure how far to go for the change but agree with your suggestions so will work them through

Reply:
I think I would also put it into a property that defaults to 60000, and then overridden to 100 in tests, I think 100 should be ok in prod but not sure

Reply:
This might be where I'm missing some context on how the API runs in prod as the code path for this connection string feels like one that would only ever be exercised either by tests or by running the app locally and not specifying the `MONGODB_USER` and `MONGODB_PASSWORD`. 

I think the tests don't actually use this string, at least the ones that extend `BaseTest`, as they define the `MONGO_DB` system property which is used instead.

Reply:
that is true, ok fair enough

Reply:
oh merged :) I'll still action Martijn's comments to improve the intent of that connection string and put that up as a new PR